backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/CloneImageGroupVolumesStructureCommand.java

```diff
@@ -220,7 +226,8 @@ private Long determineImageInitialSize(Image sourceImage,
 
         Guid hostId = imagesHandler.getHostForMeasurement(storagePoolId, imageGroupID);
         if (hostId != null) {
-            if ((srcDomainStorageType.isBlockDomain() || !sourceImage.isActive()) &&
+            boolean isPlugged = isAttachedToRunningVMs(imageGroupID) && sourceImage.isActive();
```

Comment:
better to check `sourceImage.isActive()` first before going to the database and how about isPluggedToRunningVm?